ostd/src/mm/kspace/kvirt_area.rs

```diff
@@ -45,29 +42,30 @@ impl AllocatorSelector for Untracked {
 
 /// Kernel Virtual Area.
 ///
-/// A tracked kernel virtual area (`KVirtArea<Tracked>`) manages a range of memory in
-/// `TRACKED_MAPPED_PAGES_RANGE`. It can map a inner part or all of its virtual memory
-/// to some physical tracked pages.
+/// A tracked kernel virtual area ([`KVirtArea<Tracked>`]) manages a range of
+/// memory in [`TRACKED_MAPPED_PAGES_RANGE`]. It can map a inner part or all of
```

Comment:
a inner part of all of --> a portion or the entirety of its virtual memory

The next paragraph should be revised similarly.